assets/blocks/build/block.json

```diff
@@ -7,6 +7,12 @@
   "category": "widgets",
   "icon": "testimonial",
   "description": "Show a list of available tours.",
+  "attributes": {
+    "noToursText": {
+      "type": "string",
+      "default": "There are no tours available."
+    }
+  },
   "example": {},
```

Comment:
Can we remove this, since it's empty?

Reply:
I added an explanation text like this:
![Screenshot 2023-11-16 at 11 30 26](https://github.com/Automattic/tour/assets/203408/097aaace-8782-4838-8ccf-9e789e9f41ff)